docs/api-inference/_toctree.yml

```diff
@@ -0,0 +1,17 @@
+- sections:
+  - local: index
+    title: Serverless Inference API
+  - local: overview
+    title: Overview
+  - local: getting_started
+    title: Getting Started
+  - local: rate_limits
+    title: Rate Limits
```

Comment:
I feel that we have too many pages here with similar or related content. Here's a sum up of current state:
- `index.md` => the landing page
  - what is Inference API / why using it / key benefits
  - what to expect from the docs
  - self promoting us
- `overview.md` => 
  - main features
  - eligibility
  - CPU vs GPU (to remove?)
  - Inference for PRO
  - FAQ
     - private models
     - autotrain (to remove?)
- `getting_started.md` => what will be the content?
- `rate_limits.md`

Reply:
`index.md` / `overview.md` is a bit redundant IMO. Also I think some topics could fall into the FAQ, making it a bit bigger and coherent (1. which models ? 2. and with PRO ? 3. and private ones?). What do you think of:

1. moving `overview.md#main-features` to `index.md` intro
2. keeping `index.md` more or less the same
3. removing "CPU vs GPU" and "autotrain" sections
4. moving FAQ to a standalone file with:
    - which models are eligible ?
    - what do I get with a PRO sub?
    - can I run private models ?
    - what are the rate limits ?
5. deleting `overview.md` and `rate_limits.md` (content already in other pages)
6. `getting_started.md` => depends what more we have

?